src/main/java/net/sf/jabref/importer/ImportMenuItem.java

```diff
@@ -219,34 +224,66 @@ public void update() {
                             }
                         }
 
+                        mainLoop:
```

Comment:
can you please solve this without such labels? they really complicate maintainability.